packages/core/src/checkout-buttons/strategies/amazon-pay-v2/errors/amazon-pay-v2-config-creation-error.ts

```diff
@@ -0,0 +1,13 @@
+import { StandardError } from '../../../../common/error/errors';
+
+export default class AmazonPayV2ConfigCreationError extends StandardError {
+    constructor(message?: string) {
+        super(
+            message ||
+                'An unexpected error has occurred during config creation process. Please try again later.',
```

Comment:
Have you looked at translations for this?

Reply:
We haven't translations for this message. Should we have?
As far as i understood it is not necessary

Reply:
i think it would be better to do it in another PR if you don't mind
this message is most for developers than for customers